src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -25,8 +17,8 @@ import {COLORS} from '../../../utils/theme';
 
 const NavHeader = (props) => {
   const {
-    isLoading,
-    initialData,
+    hideSeeMember,
+    animatedHeight,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of `isLoading` and `initialData` props might cause issues if they were used in the component. Please ensure that these props are not required elsewhere in the component or by any components that use `NavHeader`.



<!-- This is an auto-generated comment by OSS CodeRabbit -->